README.md

```diff
@@ -14,3 +11,32 @@
 [![Imports: isort](https://img.shields.io/badge/%20imports-isort-%231674b1?style=flat&labelColor=2e3a44)](https://pycqa.github.io/isort/)
 [![Checked with mypy](http://www.mypy-lang.org/static/mypy_badge.svg)](http://mypy-lang.org/)
 [![Code Style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg)](https://github.com/prettier/prettier)
+
+
+_A collection of packages used to process imagery_
+
+# Packages
+## [1. topo-imagery](https://github.com/linz/topo-imagery/tree/master/containers/gdal)
+
+### GDAL Container
```

Comment:
Same here, should we rename this title as `Container description` or something like that?